python/ml4ir/applications/ranking/tests/data/configs/model_config_reduce_lr_on_plateau.yaml

```diff
@@ -0,0 +1,15 @@
+architecture_key: linear
+layers:
+  - type: dense
+    name: linear_layer
+    units: 1
+    kernel_initializer: Ones
+    use_bias: False
+    activation:
+optimizer:
+  key: sgd
+  gradient_clip_value: 100.0
+lr_schedule:
+  key: reduce_lr_on_plateau
+  learning_rate: 50.0
```

Comment:
A somewhat extreme value even for a test...!

Reply:
Sure :) but I intentionally wanted the learning rate to be extremely bad so that the optimizer hits a plateau and lr reductions kicks in.